### PR TITLE
Upload of folders

### LINKED ISSUE #4145: Upload of folders

### PR DESCRIPTION
**Pull request by xiroV**
fixes #4145

### CODE REVIEW

File: frontend-web/webclient/app/Files/FileBrowse.tsx [hunk 1]
```diff
@@ -1446,9 +1446,10 @@ function FileBrowse({opts}: {opts?: ResourceBrowserOpts<UFile> & AdditionalResou
         });
     }
 
+    // TODO(Brian): Drag'n'drop
```
Comment (Tiggles) [inline frontend-web/webclient/app/Files/FileBrowse.tsx:1449]: I think these lines can go for now

File: frontend-web/webclient/app/Files/FileBrowse.tsx [hunk 2]
```diff
@@ -1446,9 +1446,10 @@ function FileBrowse({opts}: {opts?: ResourceBrowserOpts<UFile> & AdditionalResou
...
     return <MainContainer
         main={<>
-            <div ref={mountRef} />
+            <div onDragEnter={() => console.log("foo")} ref={mountRef} />
```
Comment (Tiggles) [inline frontend-web/webclient/app/Files/FileBrowse.tsx:1452]:
```suggestion
            <div ref={mountRef} />
```

File: frontend-web/webclient/app/Files/HTML5FileSelector.ts [hunk 3]
```diff
@@ -53,61 +23,57 @@ interface FileSystemDirectoryReader {
...
-    let reader: FileSystemDirectoryReader | null | undefined = entry.createReader();
-    const stack: FileSystemDirectoryEntry[] = [];
-
-    function fetchNewReader(): boolean {
-        if (!reader) {
-            reader = stack.pop()?.createReader();
-            if (!reader) {
-                return false;
-            }
+}*/
```
Comment (Tiggles) [inline frontend-web/webclient/app/Files/HTML5FileSelector.ts:29]:
```suggestion
```

File: frontend-web/webclient/app/Files/HTML5FileSelector.ts [hunk 4]
```diff
@@ -53,61 +23,57 @@ interface FileSystemDirectoryReader {
 
 type FileListFetcher = () => Promise<PackagedFile[]>;
 
-export interface FileUploadEvent {
+/*export interface FileUploadEvent {
```
Comment (Tiggles) [inline frontend-web/webclient/app/Files/HTML5FileSelector.ts:26]:
```suggestion
```

File: frontend-web/webclient/app/Files/FileBrowse.tsx (hunk 1 above)
Comment (DanThrane) [inline frontend-web/webclient/app/Files/FileBrowse.tsx:1449]:
```suggestion
```

**Review by DanThrane — APPROVED** [review]

**Pull request merged**